pkg/instrumentation/helloworld/helloworld_hook.go

```diff
@@ -71,4 +71,12 @@ func MyHook1After(ictx inst.HookContext) {
 	println("After MyStruct.Example()")
 }
 
+func MyHookGenericBefore(ictx inst.HookContext, _, _ interface{}) {
```

Comment:
Add usages of  HookContext methods inside Hook function to exercise their functionalities